### PR TITLE
Update validationRequest model to include createdAt, updatedAt, and r…

### PR DESCRIPTION
**Pull request by 0marSalah**
…ating fields

### CODE REVIEW

File: __tests__/Intergration.test.ts [hunk 1]
```diff
@@ -83,7 +83,7 @@ describe("Integration tests", () => {
```
Comment (gvelez17) [inline __tests__/Intergration.test.ts:83]: spelling - "Integration.test.ts"

File: __tests__/Intergration.test.ts [hunk 2]
```diff
@@ -83,7 +83,7 @@ describe("Integration tests", () => {
 
     it("should get all claims if no query params provided", async () => {
       // const response = await request(app).get("/api/claim"); // not working
-      const response = await request(app).get("/api/claims-all");
+      const response = await request(app).get("/api/claimsfeed2");
```
Comment (gvelez17) [inline __tests__/Intergration.test.ts:86]: are we having Candid and LInkedTrust app both use this same backend?  I think we should make the Candid routes separate if we find we have to make any changes, or layer something on them, or pass down the search - make sure both LinkedTrust and Candid can make changes freely, not affect each other, just in general thought

File: prisma/schema.prisma [hunk 3]
```diff
@@ -159,25 +159,30 @@ model Image {
...
+  createdAt       DateTime @default(now())
+  updatedAt       DateTime @updatedAt
+
   claimId          Int?    @unique
   firstName        String?
   lastName         String?
   candid_entity_id String
   email            String
   profileURL       String
+  response         ResponseStatus?
```
Comment (gvelez17) [inline prisma/schema.prisma:171]: should response be decoupled, maybe they sent out 20 validations, have 20 responses

responseStatus should be with the request, not the user i think?

with the user maybe we have a score, i feel like it oculd be in a separate table tho?

File: prisma/schema.prisma [hunk 4]
```diff
@@ -159,25 +159,30 @@ model Image {
...
 
   context           String
   validatorName     String
   validatorEmail    String
-  claimId           Int              // the id of the candid user claim (the root claim)
-  validationClaimId Int?             @unique // the id of validation claim
+  claimId           Int
+  validationClaimId Int?             @unique
   validationStatus  ValidationStatus @default(PENDING)
-  response          ResponseStatus?
```
Comment (gvelez17) [inline prisma/schema.prisma:180]: i think response should be with the request, NOT the user, i woudl keep it here

File: prisma/schema.prisma [hunk 5]
```diff
@@ -159,25 +159,30 @@ model Image {
...
+  rating            Int?
```
Comment (gvelez17) [inline prisma/schema.prisma:185]: rating woudl go with the user, or with the user-profile connection?  it coudl also be with each request i guess how valid is taht request

File: src/dao/api.dao.ts [hunk 6]
```diff
@@ -247,70 +247,94 @@ export class NodeDao {
   };
 
   getFeedEntries = async (offset: number, limit: number, search: string) => {
```
Comment (gvelez17) [inline src/dao/api.dao.ts:249]: wait wait wait is this a general getFeedEntries used by LinkedTrust?

I was making changes to this also!

please make a separate endpoint for Candid so you can work quickly without breaking LinkedTrust

we can see later if we can refactor them together.  please keep separate for now

File: src/dao/api.dao.ts [hunk 7]
```diff
@@ -247,70 +247,94 @@ export class NodeDao {
...
+      const feedEntries = await prisma.$queryRaw<FeedEntry[]>(rawQ);
+
+      const claimIdMap = new Map<number, FeedEntry>();
+      feedEntries.forEach((entry) => {
+        if (!claimIdMap.has(entry.claim_id)) {
+          claimIdMap.set(entry.claim_id, entry);
+        }
+      });
+
+      const uniqueEntriesByClaimId = Array.from(claimIdMap.values());
```
Comment (gvelez17) [inline src/dao/api.dao.ts:322]: this 100% should not be necessary i don't understand why this is here, you can only have 1 claim id, something must be wrong with the query

File: prisma/schema.prisma (hunk 5 above)
Comment (0marSalah) [reply prisma/schema.prisma:185]: rating is for validation and we use it to get the response color

File: prisma/schema.prisma (hunk 4 above)
Comment (0marSalah) [reply prisma/schema.prisma:180]: hmm that thats depend on where we return the response is it for the candid user or for each request?
in the proposal doc we return the colors on the claimant (the candid user I guess)

File: src/dao/api.dao.ts (hunk 6 above)
Comment (0marSalah) [reply src/dao/api.dao.ts:249]: wow this is not should be here lemme find out who made these changes because I didn't

**Pull request closed without merging**